### PR TITLE
New category

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR looks bigger than it is.
The new code is updating build_sample to also include application tag category to the sample collection. This is done by reading the application tag from the lims sample and looking for the category in the application tag collection.

The following 4 files have changes of importance:
**vogue/adapter/plugin.py** - new function added to get category from apptag.
**vogue/build/lims.py** - adapter added as argument and adapter.get_category is called from within build_sample
**vogue/load/sample.py** - adapter aded as argument when calling build_sample
**tests/build/test_build_sample.py** - tests added and updated

### CODE REVIEW

File: tests/build/test_build_sample.py [hunk 1]
```diff
@@ -1,18 +1,71 @@
...
-def test_build_sample(lims_sample, lims):
-    ## GIVEN a lims sample
     ## WHEN building a mongo sample
-    lims_sample.udf['Sequencing Analysis'] = 'WGSLIF'
-    mongo_sample = build_sample(lims_sample, lims)
+    mongo_sample = build_sample(lims_sample, lims, adapter)
+    
+
     ## THEN the sample should have been parsed in the correct way
     assert mongo_sample['_id'] == lims_sample.id
```
Comment (moonso) [inline tests/build/test_build_sample.py]: It would be nice to test if the correct category was added here

Comment (moonso) [inline tests/build/test_build_sample.py]:
```suggestion
    assert mongo_sample['_id'] == lims_sample.id
    assert mongo_sample['category'] == 'wgs'
```

Comment (mayabrandi) [reply tests/build/test_build_sample.py]: 👍

**Comment by moonso** [comment]
BTW I really like the PR summary that you did!

**Comment by mayabrandi** [comment]
Okej now @moonso ?

**Pull request merged**